web/src/components/Header.js

```diff
@@ -40,7 +38,13 @@ class Header extends Component {
   };
 
   toggleDropdown = () => {
-    this.setState(prev => ({ ariaExpanded: !prev.ariaExpanded }));
+    this.setState(prev => {
+      if (!prev.ariaExpanded) {
+        // add global click handler when the dropdown is expanded
+        document.addEventListener('click', this.handleOutsideClick);
```

Comment:
🙌 I didn't realize I could add a global event listener outside of `componentDidMount` -- thank you!